iofog-agent-daemon/src/main/java/org/eclipse/iofog/field_agent/FieldAgent.java

```diff
@@ -696,7 +703,7 @@ private JsonArray loadMicroservicesJsonFile() {
      *
      * @param fromFile - load from file
      */
-    private List<Microservice> loadMicroservices(boolean fromFile) {
```

Comment:
Why?

Reply:
By mistake, space is added. Will remove it